scripts/submit_sim_imcalc.py

```diff
@@ -111,7 +111,7 @@ def submit_im_calc_slurm(
 
         # create temporary station list if "match_obs_stations" is directory
         if path.isdir(
-            params[const.SlBodyOptConsts.advanced_IM.value]["match_obs_stations"]
+            str(params[const.SlBodyOptConsts.advanced_IM.value]["match_obs_stations"])
```

Comment:
Why does this need to be cast to a str?

Reply:
isdir crashes if the variable is not a path/string format. casting it to str will allow the value to be None/null.